flang/test/Driver/fveclib-codegen.f90

```diff
@@ -1,6 +1,7 @@
 ! test that -fveclib= is passed to the backend
 ! -target aarch64 so that ArmPL is available
```

Comment:
While we are here, I don't quite understand this comment. @tblah , why is this referring to ArmPL? :)

Reply:
Thanks for clarifying!

> To test ArmPL we have to have an aarch64 target otherwise the driver will generate an error. 

Right, but that's not what this comment is saying:

> ! -target aarch64 so that ArmPL is available

Also, looks like this is testing LIBMVEC rather than ArmPl. 

IMHO, referring to something that's not being tested is confusing :) Let's either add ArmPl or remove this comment 🙏🏻

Reply:
Ahh I see. Sorry I had this confused with the other test which still checks ArmPL. See https://github.com/llvm/llvm-project/pull/105528